Banking/src/main/java/com/example/Banking/exception/IdNotFound.java

```diff
@@ -0,0 +1,21 @@
+package com.example.Banking.exception;
+import lombok.Data;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Component;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@Data
+@ResponseStatus(value = HttpStatus.NOT_FOUND)
+@Component
+
+public class IdNotFound extends Exception {
+
+    private static final long serialVersionUID = 1L;
+
+    private String msg;
+    public IdNotFound() { }
+
+    public IdNotFound(String err) {
```

Comment:
You can use `@AllArgsConstructor` at the top of the class to avoid creating constructor by yourself

Reply:
I have used @AllArgsConstructor and pushed the code